geowebcache/geowebcache.properties

```diff
@@ -9,3 +9,11 @@
 # Context path of the application
 # default: /geowebcache
 #contextPath=/geowebcache
+
+# Directory of the geowebcache.xml configuration file
+# Note: you SHOULD uncomment and set the value to a persistent directory.
+# If you let the configuration file path to /tmp/gwc_config/geowebcache.xml,
+# it will be deleted and recreated from default configuration template at
+# reboot. Consider something like /etc/geowebcache.
+# default: /tmp/gwc_config
+#configurationDir=/tmp/gwc_config
```

Comment:
Since it is meant to store a configuration file, why not using`/etc/georchestra/geowebcache/` instead ?

Reply:
I copied what we have done for geonetwork (see https://github.com/georchestra/datadir/blob/18.06/geonetwork/geonetwork.properties#L2). And using `/tmp` allows to avoid unwanted side-effects when testing, because the tests are removed after rebooting, so we are not doing breaking changes (the default cache directory for geowebcache is also in `/tmp` currently)
But I'm OK if we change it to `/etc/georchestra/geowebcache` as most users will have to modify instead.

Do I change?

Reply:
Let's try it, yes.
Eventually with a sub-dir like /etc/georchestra/geowebcache/config ?

Reply:
sorry for nitpicking, but... generally /etc/georchestra (and /etc/georchestra/geowebcache) will be generated with a git clone of datadir, so we will be mixing files from datadir and XML configuration files. I don't know what is the best way (or the most common way in geOrchestra instances) to manage the georchestra, geonetwork and geoserver datadirs.

Reply:
okay, you're right that it's a different kind of configuration file ... more similar to the geonetwork and geoserver datadir.
Mixing both would probably create more issues than it solves.
Let's think about it twice.

Reply:
OK, I made my mind, let's keep it like that for the general case.
The docker branches will adopt a different setting for gwc's `configurationDir`

Reply:
Thanks for nitpicking ;-)